app/schemas/event.py

```diff
@@ -1,91 +1,43 @@
-from typing import Optional, Union, Dict, Any
-from enum import Enum
-from starlette.websockets import WebSocket
-from pydantic import BaseModel
-from app.api.api_v1.endpoints.flight_calculator import GeoCoordinates
-
-
-class JoinMode(str, Enum):
-    online = "online"
-    in_person = "in_person"
-
-
-UID = str
+from typing import Optional
 
+from pydantic import BaseModel
 
-class ParticipantModel(BaseModel):
-    location: Union[GeoCoordinates, Dict[str, str]]
-    join_mode: JoinMode
-    websocket: Any
-    active: bool = True
-    uid: Optional[UID] = None
 
+# Shared properties
+class EventBase(BaseModel):
+    name: Optional[str] = None
+    lon: Optional[float] = None
+    lat: Optional[float] = None
 
-class EventModelWebsocket:
```

Comment:
@NicolasZanotti the problem is that I've removed EventModelWebsocket but not pushed the changes that replace it. Sorry about that! Do you have time for a very short call about a design decision I was mulling over here?

Reply:
I've added the class back in for the time being, which should mean this PR is ready to land. We can talk more about the websocket/request flow going forwards in the meeting and make changes separately.